rasa/core/channels/socketio.py

```diff
@@ -145,6 +147,9 @@ def __init__(
         self.namespace = namespace
         self.socketio_path = socketio_path
 
+    def get_output_channel(self) -> Optional["OutputChannel"]:
+        return SocketIOOutput(SocketIOInput.sio , None, self.bot_message_evt)
```

Comment:
I think we should handle the case where `SocketIOInput.sio` is `None` (e.g. if the Rasa Open Server uses multiple Sanic workers (which equals multiple processes))

Reply:
Any suggestions on how to do it?

Reply:
How about printing a warning and returning `None`?

Reply:
@wochinge Updated with suggetions